Jenkinsfile

```diff
@@ -16,6 +16,9 @@ pipeline {
     SCCACHE_BUCKET='logdna-sccache-us-west-2'
     SCCACHE_REGION='us-west-2'
   }
+  parameters {
```

Comment:
Added a parameter to allow us to trigger the image push on branches other than master for testing purposes

Reply:
Added a note